src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -480,12 +480,13 @@ def update_azure_storage_account(cmd, resource_group_name, name, custom_id, stor
     return result.properties
 
 
+# note: also used for logicapps
 def enable_zip_deploy_functionapp(cmd, resource_group_name, name, src, build_remote=False, timeout=None, slot=None):
```

Comment:
while the functionapp zip deploy method is re-used , I see checks for cosumption plan , does logic apps support consumption plans? I thought the plans/skus for logic apps are different from function apps? Could you please check, to ensure that the command doesn't run into weird exceptions?

Reply:
Based on these docs -- https://docs.microsoft.com/en-us/azure/logic-apps/single-tenant-overview-compare Logic Apps can be on consumption plans as well as the workflow standard plans. I'll test out some scenarios of logic apps on consumption plans to ensure the zip deploy is still functional